plugins/http_plugin/http_plugin.cpp

```diff
@@ -135,55 +123,18 @@ class http_plugin_impl : public std::enable_shared_from_this<http_plugin_impl> {
          }
 
          void create_beast_server(bool useSSL, bool isUnix=false) {
```

Comment:
Remove `bool useSSL` from the parameter list.

Reply:
Thanks Lin, done.